src/defines.ts

```diff
@@ -37,6 +37,7 @@ export interface IdentifyResult {
   text: string;
   type: StatementType;
   executionType: ExecutionType;
+  parameters?: string[];
```

Comment:
I think I'd prefer just making this a permanent field to the result. Easier to deal with on the consumer code, and I don't consider adding an additional field to the return result a breaking change.